CONTRIBUTING.md

```diff
@@ -10,6 +10,8 @@ The workflow is pretty standard:
 
 ## Running Tests
 
+Create a virtual host `rabbit/hole` and assign it guest permissions
```

Comment:
It's a better idea to recommend running [bin/ci/before_script.sh](https://github.com/michaelklishin/rabbit-hole/blob/master/bin/ci/before_build.sh).